spanner/benchmark/bin/ycsb

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
This is file necessary? Can't you just run the Python script? Or add a new session to `nox`?

Reply:
This is a YCSB-like executable that can be integrated into PerfKitBenchmarker. Will add a comment.

Reply:
Sorry, but does this file also need a license header?

Reply:
Done